### ISSUE TITLE
Не работает "addLead" для сущности contact

### ISSUE THREAD

**Issue by ikulakov** [drillcoder/AmoCRM_Wrap#13]
Обе сущности получены, но привязка не происходит. Если скорректировать код под предыдущую 6 версию, то работает нормально

**Comment by maler1988** [comment]
Да, там почему-то в методе saveBase() класса BaseEntity при вызове $entity->load($res->_embedded->items[0]->id) теряются id сущности. В частности пробовал после сохранения контакта добавить его же в лид при помощи addContact().

**Comment by AlexMosiakin** [comment]
> Да, там почему-то в методе saveBase() класса BaseEntity при вызове $entity->load($res->_embedded->items[0]->id) теряются id сущности. В частности пробовал после сохранения контакта добавить его же в лид при помощи addContact().

Удалось ли как-то решить проблему?

**Comment by AlexMosiakin** [comment]
> Обе сущности получены, но привязка не происходит. Если скорректировать код под предыдущую 6 версию, то работает нормально

А что именно корректировали? Пытался поправить под 6 версию, но особого результата не получил

**Comment by ikulakov** [comment]
Честно говоря уже не вспомню, перешел на официальную библиотеку от амо. Написана более мудрёно чем эта, но работает вроде как по стабильнее и с последней версией API

**Comment by AlexMosiakin** [comment]
> Честно говоря уже не вспомню, перешел на официальную библиотеку от амо. Написана более мудрёно чем эта, но работает вроде как по стабильнее и с последней версией API

Понял, спасибо за ответ!

**Comment by Dmi3yy** [comment]
Все работает, столкнулся с той же проблемой. 

Нужно было банально сохранить в переменную: 
$lead  =  $lead = $lead->save(); тогда будет id и будет все ок
